engine/src/gfx/jpeg_memory.h

```diff
@@ -24,7 +26,7 @@
 #endif //tells VCC not to generate min/max macros
 #include <windows.h>
 #endif
-#include "vsfilesystem.h"
+#include "../vsfilesystem.h"
```

Comment:
Are you sure about this? relative path in include is not the best practice... do you see any issues in compilation due to this?

Reply:
@nabaco , I'm not sure of anything at this point. My compiler does not complain about missing headers, it was just my IDE. However, with a little help from Stephen Tuggy I may have sorted out the IDE as well.